editing-this-document.md

```diff
@@ -0,0 +1,32 @@
+# Editing This Document
+
+OpenTx is a community driven project. We appreciate your willingness to help.
+
+To help edit this document please follow these steps.
+
+## GitHub Fork
+
+* Sign in to GitHub
+* Go to [https://github.com/opentx/](https://github.com/opentx/)
+* Select &gt; lua-reference-guide
+* Click on Fork \(top right of the screen\)
+
+## GitBook Link
+
+* Sign in to GitBook  [https://www.gitbook.com/login](https://www.gitbook.com/login) \(Sign in with GitHub\)
+* Account Settings &gt; Profile &gt; GitHub &gt; With Access to public repositories &gt; Authorize permissions
+* Create a new Book & Manual &gt; Title "OpenTx Lua Reference"&gt; Create Book
+* Book's Settings &gt; GitHub &gt; Select a Repository &gt; Sync
+* _An error message will display, don't worry_
+* Select Force sync using GitHub content
+  Example success message "This book is linked to the GitHub repository:[RCdiy/lua-reference-guide](https://github.com/RCdiy/lua-reference-guide)"
+
+## Editing
+
+* You can now click on the blue Edit button \(top right\)
+* Edit in GitBook &gt; Publish
+* Go to GitHub &gt; Compare & Pull Request
+  When you submit a pull request please add descriptive notes \(not the default you will find filled in\)
+
+
+
```

Comment:
Do we want this kind of info in the actual book or not? In my opinion this kind of info should go into the wiki pages, not end user documentation. But I may be wrong.